operator/v1/types.go

```diff
@@ -10,32 +10,32 @@ import (
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:scope=Cluster
 
-// HubCore represents a deployment of nucleus hub core component.
-// HubCore will be only deployed in open-cluster-management-hub namespace.
-type HubCore struct {
+// ClusterManager configures the controllers that govern registration and work distribution for attached klusterlets.
```

Comment:
maybe `ClusterManager configures the controllers on hub`?

Reply:
Thinking about this one.